src/ProductTour/Checkpoint.jsx

```diff
@@ -95,8 +96,13 @@ const Checkpoint = React.forwardRef(({
       role="dialog"
       style={{ visibility: checkpointVisible ? 'visible' : 'hidden', pointerEvents: checkpointVisible ? 'auto' : 'none' }}
     >
-      {/* This text is not translated due to Paragon's lack of i18n support */}
-      <span className="sr-only">Top of step {index + 1}</span>
+      <span className="sr-only">
+        <FormattedMessage
+          id="pgn.checkpoint.sr-only.message"
+          defaultMessage={`Top of step ${index + 1}`}
```

Comment:
Do I need to add keys for this translation to the [i18n](https://github.com/openedx/paragon/tree/master/src/i18n/messages) folder?